### DISCUSSION TITLE
"argon" pro not having a slider sliding sound that other skins have

### DISCUSSION THREAD

**Discussion by soopax** [ppy/osu#23840] [Q&A] [answered]
1. Import https://osu.ppy.sh/beatmapsets/1405913#osu/2916770 
2. In the "EASY" difficulty go to 00:26:450 (1). 

This slider has a whistle sound when using 
* 'osu! "argon" (2022) (team osu!)'
* 'osu! "triangles" (2017) (team osu!)'
* 'osu! "classic" (2013) (team osu!)'

But I can't hear the whistle sound in 'osu! "argon" pro (2022) (team osu!)'. No sound for hitting the slider feels bad to me.

**Comment by peppy** [comment]
I believe this was an intentional decision by @nekodex, maybe he can provide a response here.

**Reply by nekodex** [reply, marked as answer]
Yeah, this was intentional.

I received feedback that pro players don't like slider slide and slider whistle sounds, thus argon pro doesn't have them.
